problems/objects.js

```diff
@@ -61,7 +71,10 @@ const appleSet = () => {};
  *
  */
 
-const adamAndEveApples = () => {};
+const adamAndEveApples = appleCountByName => {
+  sum = appleCountByName.Adam + appleCountByName.Eve
```

Comment:
There should be a let in front of the sum variable in order to declare it. 
However, you don't necessarily need a variable to return the number here.